api/services/locationService.ts

```diff
@@ -0,0 +1,66 @@
+import { makeSCOSRequest } from './scosService';
+import getLogger from '../utils/logger';
+import { AGENCIES_TABLE } from '../utils/constants';
+import LocationDto from '../models/dto/locationDto';
+import ScosAgencyDto from '../models/scosApi/scosAgencyDto';
+import { notEmpty } from '../utils/typeGuards';
+
+const log = getLogger('locationService');
+
+function mapToLocationDto(agency: ScosAgencyDto): LocationDto | null {
+  // "Site" is an array of 1 element
+  const site = agency.site_info.site[0];
+
+  if (!site) {
+    // there's no location data
+    return null;
+  }
+
+  // as far as I can tell, "Address" has 2 elements but the second is a throwaway
+  const address = site.address[0];
+  const address1 = address.line1;
+  const address2 = `${address.line2} ${address.line3}`.trim();
+
+  // not sure the difference between "st" vs "stg" phones
+  // so we'll just pick the first one that exists
+  const phoneList = [...site.stgphones, ...site.stphones];
+  const phoneNumber = phoneList[0]?.phone ?? '';
```

Comment:
Are we able to join them with commas separating instead of picking the first? We may need to display multiple numbers in case someone can only call 800 numbers or one of the numbers is no longer in service

Reply:
```js
const phoneNumber = phoneList.map(item => item.phone).join(', ') || '';
```

Reply:
This would be a question for Hands On - do we have a contact over there to ask?

Reply:
For some context, the values in these fields are things like `(614) 645-7250 Administrative` - but have a bunch of different labels.

For some locations, the stgphones & stphones lists are the same, but for some they're different. (I'm not even sure what `st` and `stg` mean in this context)

We could potentially return a unique-d array to the frontend, and display all of it. Or we could talk to someone who actually knows the dataset, but I'm not sure who that is :)